databricks/koalas/missing/series.py

```diff
@@ -156,6 +156,10 @@ class _MissingPandasLikeSeries(object):
     real = unsupported_property(
         'real',
         reason="If you want to collect your data as an NumPy array, use 'to_numpy()' instead.")
+    nbytes = unsupported_property(
+        'nbytes',
+        reason="Since nbytes trigger whole computation to get the size, which is pretty "
+               "critical when you handle large dataset, we don't support this for now")
```

Comment:
@itholic, I think we can mention like check Spark UI to monitor memory usage or shuffle IO. Also, please avoid abbreviation like `don't` in public error message or documentation.

Reply:
oh i understand it. Thanks for comment !